internal/server/errors.go

```diff
@@ -52,9 +53,15 @@ func sendAPIError(c *gin.Context, err error) {
 	case errors.As(err, &uniqueConstraintError):
 		resp.Code = http.StatusConflict
 		resp.Message = err.Error()
+		// remove the error trace from field error message
```

Comment:
This prevents the error in the UI from displaying the full trace.

Ex:
An error message of `sign-up: create org: duplicate domain` becomes `duplicate domain`